test/langtools/tools/javac/doctree/AtEscapeTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8300914
+ * @summary Allow `@` as an escape in documentation comments
+ * @modules jdk.compiler/com.sun.tools.javac.api
+ *          jdk.compiler/com.sun.tools.javac.file
+ *          jdk.compiler/com.sun.tools.javac.tree
+ *          jdk.compiler/com.sun.tools.javac.util
+ * @build DocCommentTester
+ * @run main DocCommentTester AtEscapeTest.java
+ */
+
+class AtEscapeTest {
+    /**
+     * abc
+     * @@tag
```

Comment:
I'd add a couple of tests to assert that `@@` and `@*` also work in the middle of a line. Ulike `@/`, which requires preceding `*`, `@@` and `@*` do not require preceding whitespace. That seems useful. In particular, `@@` could be used in the middle of a line for robustness. For example, one could prepend plaintext `@Override` in the middle of a line with an extra `@` to defend against text reflow that brings `@Override` to the beginning of a line.

Reply:
Will do.

Reply:
Note that `@@` and `@*` are also context-sensitive, and can only be used where the unescaped character would have a different effect.

Nevertheless, the suggestion for additional tests to demonstrate the behavior is well-taken.

Reply:
done